auth/src/main/java/org/aerogear/mobile/auth/utils/SanityCheck.java

```diff
@@ -0,0 +1,17 @@
+package org.aerogear.mobile.auth.utils;
+
+public class SanityCheck {
```

Comment:
actually, this is useful for all the other modules so I think we should move this class to the core module.

Reply:
Agree. Will move it.